apps/lib/cmp_mock_srv.c

```diff
@@ -391,6 +408,46 @@ static OSSL_CMP_PKISI *process_rr(OSSL_CMP_SRV_CTX *srv_ctx,
     return OSSL_CMP_PKISI_dup(ctx->statusOut);
 }
 
+/* return -1 for error, 0 for no update available */
+static int check_client_crl(const STACK_OF(OSSL_CMP_CRLSTATUS) *crlStatusList,
+                            const X509_CRL *crl)
+{
+    OSSL_CMP_CRLSTATUS *crlstatus;
+    DIST_POINT_NAME *dpn;
```

Comment:
```suggestion
    DIST_POINT_NAME *dpn = NULL;
```

Reply:
This variable get assigned by `OSSL_CMP_CRLSTATUS_get0`, but for code readability I will initialize it with NULL.

Reply:
Sure. I don't see it as incorporated yet but it is marked as resolved?

Reply:
Please see the commit 2b55a726384e96ef1a9f75f4aa9de0c929ad9e90